### ISSUE TITLE
Cutover Worlds of Change (formerly CNA) to CURIOSity

### ISSUE THREAD

**Issue by VanessaVenti** [harvard-lts/CURIOSity#338]
# Description
Colonial North America (CNA) is being renamed to Worlds of Change (WOC) and migrated from its own standalone Spotlight instance to CURIOSity's Spotlight instance.

## Add CNA redirect to id.lib
When DPLA harvested the CNA set, they used the https://id.lib.harvard.edu/cna/ OICs that point to the current site. This redirect needs to be in place before we publish the new site and shut down the current one so that we don't break the [32K+ links in DPLA](https://dp.la/search?collection=%22Colonial+North+America+at+Harvard+Library%22). Here are examples of the old and new OICs:
- [Record in DPLA](https://dp.la/item/33016be4df92800f616393279fc2a03a?q=Roolle+de+la+monstre+et+reveue+faicte+en+armes+en+la+basse+court+du+chasteau+de+Calumet) (with the old CNA OIC)
- Old CNA OIC: https://id.lib.harvard.edu/cna/990092051840203941
- New WOC OIC: https://id.lib.harvard.edu/curiosity/worlds-of-change/149-990092051840203941

## Steps
- [x] Add a redirect rule in the id.lib app to redirect the old CNA syntax to the new curiosity "Worlds of Change" syntax
- [x] Make the change to dev and qa (on feedback-dev.lib.harvard.edu), and test
- [x] Retest idtest.lib.harvard.edu:9020/cna/990092051840203941 (and other examples)
- [x] Anna will get go ahead from Claire D. to cutover the site
- [x] Deploy to prod (feedback-prod.lib.harvard.edu). This needs to happen during a maintenance window (Tue and Thu before 7) and in concert with the WOC site cutover. VV and AVS will give MV at least week's notice of the cutover date for WOC.
- [x] Write to Clare O'Keeffe / Kerry Conley about process for changing DNS https://colonialnorthamerica.library.harvard.edu to resolve to https://curiosity.lib.harvard.edu/worlds-of-change (Enrique)
- [ ] Decommission the services hosting the original CNA site (LTS and ProdOps)

## Other steps
- [x] Once the redirect is in place, we can delete the old CNA set that contains the cna id.lib so those OICs no longer appear in LibraryCloud. They will still be in DPLA and work until our next recontribution where they will then grab the new curiosity id.lib OICs instead. (Vanessa)

**Comment by michael-lts** [comment]
@VanessaVenti,
On the id.lib qa server, I changed the redirect from the old id.lib to go to the new worlds of change site, if you want to test with this and other id examples
http://idtest.lib.harvard.edu:9020/cna/990092051840203941

**Comment by VanessaVenti** [comment]
I tested this with a few IDs and all point to the new site. The new site is not live yet, so we'll coordinate the id.lib move to prod with WOC's cutover once we have a date set for that.

**Comment by VanessaVenti** [comment]
@dl-maura and @enriquediaz the only item unchecked above is "Decommission the services hosting the original CNA site (LTS and ProdOps)". Is that what will happen a month after cutover or was that already done? Just wondering if we can check of that item and close out the story :)

**Comment by enriquediaz** [comment]
That's right @VanessaVenti the data will be purged after 30 days, which is on July 17th, and then we can close this out.

**Comment by dl-maura** [comment]
@VanessaVenti can we close?

**Comment by VanessaVenti** [comment]
@dl-maura - I think we were just waiting on "Decommission the services hosting the original CNA site (LTS and ProdOps)" which @enriquediaz says should have happened on July 17th. Do you know if that happened? If so, we can close this ticket.
